File: solutions/csharp/pangram/1/Pangram.cs
using System;
using System.Collections.Generic;
using System.Linq;

public static class Pangram
{
  private static char[] alphabet = {
    'a',
    'b',
    'c',
    'd',
    'e',
    'f',
    'g',
    'h',
    'i',
    'j',
    'k',
    'l',
    'm',
    'n',
    'o',
    'p',
    'q',
    'r',
    's',
    't',
    'u',
    'v',
    'w',
    'x',
    'y',
    'z'
  };

  public static bool IsPangram(string input)
  {
    return alphabet.All(letter => input.ToLower().Contains(letter));
  }
}

File: solutions/fsharp/pangram/1/Pangram.fs
module Pangram

open System

let alphabet = Set.ofList [ 'a' .. 'z' ]

let isPangram (input: string): bool =
    input
      |> Seq.toList
      |> List.map Char.ToLower
      |> Set.ofList
      |> fun inputLetters -> alphabet - inputLetters
      |> fun remainingLetters -> remainingLetters.Count = 0